--- ./Makefile.in.org	Tue Aug 20 03:43:36 2002
+++ ./Makefile.in	Wed Aug 28 04:41:58 2002
@@ -16,9 +16,11 @@
 	make -f Make.debug
 
 install: rssh
-	for i in $(PROGS); do echo "install -s -o root -g root -m 0555 $$i $(installdir)"; install -s -o root -g root -m 0555 $$i $(installdir); done
-	for i in $(MANPGS); do echo "install -o root -g root -m 0444 $$i $(mandir)"; install -o root -g root -m 0444 $$i $(mandir); done
-	for i in $(LINKS); do echo "ln -s rssh $(installdir)/$$i"; ln -s rssh $(installdir)/$$i; done
+	install -d $(DESTDIR)$(installdir)
+	install -d $(DESTDIR)$(mandir)
+	for i in $(PROGS); do echo "install -s -m 0555 $$i $(installdir)"; install -s -m 0555 $$i $(DESTDIR)$(installdir); done
+	for i in $(LINKS); do echo "ln -s rssh $(installdir)/$$i"; ln -s rssh $(DESTDIR)$(installdir)/$$i; done
+	for i in $(MANPGS); do echo "install -m 0444 $$i $(mandir)"; install -m 0444 $$i $(DESTDIR)$(mandir); done
 
 uninstall:
 	for file in $(PROGS); do rm -f $(installdir)/$$file; done
