--- ./Makefile.in.org	Wed Sep 18 06:28:31 2002
+++ ./Makefile.in	Wed Sep 18 23:28:46 2002
@@ -1,18 +1,20 @@
-installdir = @prefix@/bin
-mandir = @prefix@/share/man/man1
-tardir = @PACKAGE_NAME@-@PACKAGE_VERSION@
-tarname = @PACKAGE_NAME@-@PACKAGE_VERSION@.tgz
-
-CC = @CC@
-CFLAGS = -O2 @static@ -Wall -Wshadow
-
-INSTALL = @INSTALL@
-
-SRC = rssh.c log.c
-OBJS = $(SRC:.c=.o)
-PROGS = rssh
-LINKS = scpsh sftpsh
-MANPGS = rssh.1 scpsh.1 sftpsh.1
+PREFIX	= @prefix@
+BINDIR	= @bindir@
+MANDIR	= @mandir@
+mandir	= @prefix@/share/man/man1
+tardir	= @PACKAGE_NAME@-@PACKAGE_VERSION@
+tarname	= @PACKAGE_NAME@-@PACKAGE_VERSION@.tgz
+
+CC	= @CC@
+CFLAGS	= -O2 @static@ -Wall -Wshadow
+
+INSTALL	= @INSTALL@
+
+SRC	= rssh.c log.c
+OBJS	= $(SRC:.c=.o)
+PROGS	= rssh
+LINKS	= scpsh sftpsh
+MANPGS	= rssh.1 scpsh.1 sftpsh.1
 CLEANFILES = core *.o $(PROGS) 
 DISTCLEANFILES = pathnames.h Makefile rssh.c config.log config.status
 DISTFILES = INSTALL Make.debug Makefile.in README SECURITY TODO configure\
@@ -27,14 +29,15 @@
 	make -f Make.debug
 
 install: rssh
-	@for i in $(PROGS); do echo "$(INSTALL) -s $$i $(installdir)"; $(INSTALL) -s $$i $(installdir); done
-	@for i in $(MANPGS); do echo "$(INSTALL) $$i $(mandir)"; $(INSTALL) $$i $(mandir); done
-	@for i in $(LINKS); do echo "ln -s rssh $(installdir)/$$i"; ln -s rssh $(installdir)/$$i; done
+	$(INSTALL) -d $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)/man1/
+	@for i in $(PROGS); do echo "$(INSTALL) -s $$i $(installdir)"; $(INSTALL) -s $$i $(DESTDIR)$(BINDIR); done
+	@for i in $(MANPGS); do echo "$(INSTALL) $$i $(mandir)"; $(INSTALL) $$i $(DESTDIR)$(MANDIR)/man1/; done
+	@for i in $(LINKS); do echo "ln -s rssh $(installdir)/$$i"; ln -s rssh $(DESTDIR)$(BINDIR)/$$i; done
 
 uninstall:
-	for file in $(PROGS); do rm -f $(installdir)/$$file; done
-	for file in $(LINKS); do rm -f $(installdir)/$$file; done
-	for file in $(MANPGS); do rm -f $(mandir)/$$file; done
+	for file in $(PROGS); do rm -f $(BINDIR)/$$file; done
+	for file in $(LINKS); do rm -f $(BINDIR)/$$file; done
+	for file in $(MANPGS); do rm -f $(MANDIR)/man1/$$file; done
 
 clean:
 	rm -f $(CLEANFILES)
